.github/actions/build/action.yml

```diff
@@ -25,4 +25,15 @@ runs:
         mkdir dist
         mklink /j .\dist\bin .\x64\Release\
       shell: cmd
-    # ------------ End Windows Build ---------------
\ No newline at end of file
+    # ------------ End Windows Build ---------------
+    # ------------ Windows Build With EXEC_ENV_OLS and USE_BING_INFRA ---------------
+    - name: Add VisualStudio command line tools into path
```

Comment:
what checks did you intend to trigger with it? Just a build check? nothing? some run checks?

Reply:
Just build, I think many of the tests are expected not to pass due to different interfaces